### PR TITLE
refact(deps): bump k8s and client-go deps to version v0.20.2

### PR DESCRIPTION
**Pull request by prateekpandey14**
Signed-off-by: prateekpandey14 <prateek.pandey@mayadata.io>

**What this PR does**:
- Bump k8s and client-go version to use v0.20.2 version
- Bumo openebs/api v2.3.0 verison

**Special notes for your reviewer**:

**Checklist**
- [ ] PR messages has document related information
- [ ] Labelled this PR & related issue with `documentation` tag
- [ ] PR messages has breaking changes related information
- [ ] PR messages has upgrade related information
- [ ] Labelled this PR & related issue with `requires-upgrade` tag
- [ ] Tests updated

### CODE REVIEW

File: pkg/iscsi/iscsi.go [hunk 1]
```diff
@@ -54,10 +52,10 @@ type iscsiDisk struct {
...
 }
 
-type iscsiPlugin struct {
-	host        volume.VolumeHost
-	targetLocks keymutex.KeyMutex
-}
+//type iscsiPlugin struct {
+//	host        volume.VolumeHost
+//	targetLocks keymutex.KeyMutex
+//}
```
Comment (shubham14bajpai) [inline pkg/iscsi/iscsi.go:58]: Can we remove this code?

Comment (prateekpandey14) [reply pkg/iscsi/iscsi.go:58]: going to uncomment them for now

**Review by shubham14bajpai — APPROVED** [review]
LGTM

File: pkg/utils/log.go [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package utils
+
+import (
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// PodLog Format a log message with pod information
```
Comment (prateekpandey14) [inline pkg/utils/log.go:8]: going to remove this file .. got added by mistake

Comment (prateekpandey14) [reply pkg/utils/log.go]: removed

**Pull request merged**